TestRun - Password Criteria Checker/Program.cs

```diff
@@ -0,0 +1,175 @@
+﻿/** 
+
+This purpose of this program is to have the user input a password to check it for adherenace to the criteria listed under Criteria Declaration
+
+More criteria will be added as the program grows.
+
+**/
+
+// Criteria Declaration
+// This space is used to declare criteria to be followed for each given password
+string criteria1 = "[1] The password must be 9 to 25 digits long";
+string criteria2 = "[2] The password must contain atleast 1 of the following characters : !,@,#,$,%,^,&,*,(,),-,=,_,+";
+string criteria3 = "[3] The password must contain atleast 1 number character";
+string criteria4 = "[4] The password must contain atleast 1 upper case letter";

```

Comment:
Typically human readable text should be extracted out into a centralized location. This has a number of benefits.
- Removes extra bulk from your implementation files
- You can send these translation files to other non-developers to translate into different languages.

i18n is generally the tool you'd use, which has different packages available depending on what programming language you use. For Ruby, we use https://github.com/ruby-i18n/i18n, which allows for the following.

```ruby
# ruby
I18n.t("path.to.translation", min: 9, max: 25)
```

```yml
# translation file
path:
  to:
    translation: The password must be %{min} to %{max} digits long
```